packages/test/src/test-integration-update-interceptors.ts

```diff
@@ -0,0 +1,79 @@
+import { WorkflowStartUpdateInput, WorkflowStartUpdateOutput } from '@temporalio/client';
+import * as wf from '@temporalio/workflow';
+import { Next, UpdateInput, WorkflowInboundCallsInterceptor, WorkflowInterceptors } from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers-integration';
+
+const update = wf.defineUpdate<string[], [string]>('update');
+
+export async function workflowWithUpdates(): Promise<string[]> {
```

Comment:
nit: move test workflows close to tests they relate to (like right before the first test that uses it)